apps/OpenSpace/ext/launcher/src/settingsdialog.cpp

```diff
@@ -0,0 +1,407 @@
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include "settingsdialog.h"
+
+#include "profile/line.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QDialogButtonBox>
+#include <QGridLayout>
+#include <QLabel>
+#include <QLineEdit>
+#include <QMessageBox>
+#include <QPushButton>
+
+SettingsDialog::SettingsDialog(openspace::Settings settings,
+                               QWidget* parent)
+    : QDialog(parent)
+    , _startEdit(settings)
+    , _currentEdit(settings)
+{
+    setWindowTitle("Settings");
+    createWidgets();
+    loadFromSettings(settings);
+}
+
+void SettingsDialog::createWidgets() {
+    // Layout of this dialog:
+    //
+    // -------------------------------------------------------
+    // | Profile                                             |
+    // | Starting Profile:          | [oooooooooooooooooooo] |
+    // | [] Keep Last Profile                                |
+    // | Configuration                                       |
+    // | Starting Configuration:    | [oooooooooooooooooooo] |
+    // | [] Keep Last Configuration                          |
+    // | User Interface                                      |
+    // | Property Visibility        | DDDDDDDDDDDDDDDDDDDDD> |
+    // | [] Bypass Launcher                                  |
+    // | Informational text about undoing the bypass setting |
+    // | MRF Caching                                         |
+    // | [] Enable caching                                   |
+    // | Cache location             | [oooooooooooooooooooo] |
+    // |                            | <Save>        <Cancel> |
+    // -------------------------------------------------------
+     
+    QGridLayout* layout = new QGridLayout(this);
+    
+    {
+        QLabel* label = new QLabel("Profile");
+        label->setObjectName("heading");
+        layout->addWidget(label, 0, 0, 1, 2);
+
+        QLabel* conf = new QLabel("Starting Profile");
+        layout->addWidget(conf, 1, 0);
+
+        _profile = new QLineEdit;
+        _profile->setToolTip(
+            "With this setting, you can choose a profile that will be loaded the next "
```

Comment:
I somehow expected this tooltip to be on the label ("Starting Profile") rather than the input text box. Should check if others agree

The same goes for all the other QLineEdits in the menu

Reply:
Yea, I added it to the labels for all of them as well